src/modules/DressMe/PersonalDeatails/FieldAndValidation.tsx

```diff
@@ -0,0 +1,51 @@
+import { Field, useField } from "formik";
+import styled from "styled-components";
+
+export interface FieldProps {
+  placeholder: string;
+  name: string;
+  type?: string;
+  min?: string;
+  max?: string;
+  step?: string;
+}
+
+const FieldAndValidation = ({ min, max, step, type, placeholder, name }: FieldProps) => {
+  const [Field, Meta] = useField(name);
+  return (
+    <FieldContainer>
+      <PersonalField
+        min={min}
```

Comment:
consider a different way to pass the props to PersonalField.
hint: as a developer, whenever you see repetative code that's a red flag that a more concise solution is desired